scripts/cloudbuild.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+projectRoot=`dirname $0`/..
+buildProperties=$projectRoot/target/classes/build.properties
+
+# reads a property value from a .properties file
+function read_prop {
+  grep "${1}" $buildProperties | cut -d'=' -f2
+}
+
+# invoke local maven to generate build properties file
+mvn generate-resources
```

Comment:
I thought you can just do
```
mvn org.codehaus.mojo:properties-maven-plugin:write-project-properties -DoutputFile=build.properties
```
without having to define the plugin in the pom, but apparently this doesn't work because the plugin doesn't accept `outputFile` as a command line argument.